### PR TITLE
TILA-1939: Add a new permission for commenting reservations

### PR DESCRIPTION
**Pull request by saaste**
Also updated the description of `can_view_reservation` permission.

### CODE REVIEW

File: permissions/models.py [hunk 1]
```diff
@@ -56,7 +56,7 @@
     ),
     (
         "can_view_reservations",
-        _("Can create, edit and delete equipment in the whole system"),
+        _("Can view equipments in the whole system"),
```
Comment (nicobav) [inline permissions/models.py:59]: "Can view reservations in the whole system"

File: permissions/models.py [hunk 2]
```diff
@@ -87,6 +87,10 @@
         "can_view_users",
         _("Can view users in the whole system"),
     ),
+    (
+        "can_comment_reservations",
+        _("Can comment reservations"),
```
Comment (nicobav) [inline permissions/models.py:92]: perhapsies we could add the "whole system" line to here as well to be consistent and making it clear.

File: permissions/api_permissions/graphene_permissions.py [hunk 3]
```diff
@@ -353,6 +354,18 @@ def has_mutation_permission(cls, root: Any, info: ResolveInfo, input: dict) -> b
         return False
 
 
+class ReservationCommentPermission(BasePermission):
+    @classmethod
+    def has_mutation_permission(cls, root: Any, info: ResolveInfo, input: dict) -> bool:
+        pk = input.get("pk")
+        if pk:
+            reservation = get_object_or_404(Reservation, pk=pk)
+            return can_handle_reservation(
```
Comment (nicobav) [inline permissions/api_permissions/graphene_permissions.py:363]: Should we add the can_handle helper func in the can_comment helper? Wouldn't need to combine here but would be just a straight helper function what to use here.

**Review by nicobav — CHANGES_REQUESTED** [review]

File: permissions/models.py (hunk 1 above)
Comment (saaste) [reply permissions/models.py]: Fixed 🤦

File: permissions/models.py (hunk 2 above)
Comment (saaste) [reply permissions/models.py]: Added 👍

File: permissions/api_permissions/graphene_permissions.py (hunk 3 above)
Comment (saaste) [reply permissions/api_permissions/graphene_permissions.py]: Good idea! I added the helper function and simplified this ✅

**Review by nicobav — APPROVED** [review]
Nicey :cookie:

**Pull request merged**